hudi-client/src/test/java/org/apache/hudi/index/hbase/TestHBaseIndex.java

```diff
@@ -156,6 +160,46 @@ public void testSimpleTagLocationAndUpdate() throws Exception {
     }
   }
 
+  @Test
+  public void testTagLocationAndPartitionPathUpdate() throws Exception {
```

Comment:
can we also have another test where, the config value is set to false and the record is tagged w/ old partition. since we have introduced a new config, would be good to have tests covering both branches.

Reply:
Do you mean to consider the HBASE_INDEX_UPDATE_PARTITION_PATH parameter true and false test cases when testing?

Reply:
yes, I see we have a test for true, but can you also add a test for false as well.

Reply:
Okay, I will refine this use case later。